crates/cli-flags/src/lib.rs

```diff
@@ -497,6 +497,25 @@ wasmtime_option_group! {
     }
 }
 
+wasmtime_option_group! {
+    #[derive(PartialEq, Clone, Deserialize)]
+    #[serde(rename_all = "kebab-case", deny_unknown_fields)]
+    pub struct RecordOptions {
+        /// Filesystem endpoint to store the recorded execution trace (empty string "" for void endpoint)
```

Comment:
s/Filesystem endpoint/filename/ (I understand the terminology in the context of a sink for the recording stream but for a user-facing option let's keep it concrete -- "Filename for the recorded execution trace (or empty string to skip writing a file)"